test/export/test_torchbind.py

```diff
@@ -392,6 +403,52 @@ def forward(self, arg0_1, attr, arg1_1):
     return (getitem_3, add_1)""",  # noqa: B950
         )
 
+    def test_make_fx_tensor_queue_methods(self):
+        class Model(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.linear = torch.nn.Linear(3, 2)
+
+            def forward(self, tq, x):
+                tq.push(x.cos())
+                tq.push(x.sin())
+                x_cos = tq.pop() + tq.size()
+                x_sin = tq.pop() - tq.size()
+                return x_sin, x_cos, tq
+
+        mod = Model()
+        tq = torch.classes._TorchScriptTesting._TensorQueue(
+            torch.empty(
+                0,
+            ).fill_(-1)
+        )
+        tq1 = torch.classes._TorchScriptTesting._TensorQueue(
```

Comment:
mmm for this to be *really* correct, then calling the TensorQueue ctor underneath a FakeTensorMode should create a FakeTensorQueue. If this is not too difficult to do, then we should do it (and add tests for just FakeTensorMode + TorchBind objects)

Reply:
Indeed, this is possible if we could intercept the `__call__` method of "ScriptClass" (i.e. torch.classes._TorchScriptTesting._TensorQueue) and doing something there. Do you mind if add this in the follow up pr?

Reply:
ya feel free